lib/salus/formatting.rb

```diff
@@ -4,22 +4,22 @@ module Formatting
     INDENT_STRING = (' ' * INDENT_SIZE).freeze
 
     TABLE_BORDER_CHARS = {
-      top_mid:      '┬',
-      top_left:     '┌',
-      top_right:    '┐',
-      bottom_mid:   '┴',
-      bottom_left:  '└',
+      top_mid: '┬',
+      top_left: '┌',
+      top_right: '┐',
+      bottom_mid: '┴',
+      bottom_left: '└',
       bottom_right: '┘',
-      left_mid:     '├',
-      right_mid:    '┤',
-      horizontal:   '─',
-      vertical:     '│',
-      cross:        '┼'
```

Comment:
I'm actually a big fan of lining things up vertically. Can we disable the rubocop rule that's forcing this?